ng-pkgs/activation_service/package.toml

```diff
@@ -1,3 +1,19 @@
 name = "activation_service"
 is_auth = false
-is_admin = false
\ No newline at end of file
+is_admin = false
+
+
+[[servers]]
+name = "activation_root_proxy"
+domain = "gettft.testnet.grid.tf"
+letsencryptemail = "tarekr@incubaid.com"
+includes = ["default_443.chatflows_*", "default_443.auth_*", "default_443.activation*"]
```

Comment:
why are ""default_443.chatflows_*" and  "default_443.auth_*" included ?

Reply:
they aren't needed in this package, we removed them